cmds/boot/localboot/systemdboot.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
```

Comment:
I'd like to try to maintain the pretense that we can run under other kernels, which I realize we have not been good about. Anyway, at some point, this probably ought to be called 
systemdboot_linux.go